great_expectations/render/renderer/notebook_renderer.py

```diff
@@ -45,52 +45,45 @@ def _build_kwargs_string(cls, expectation):
 
         return ", ".join(kwargs)
 
-    def add_header(self, data_asset_name=None, suite_name=None):
-        # TODO better wording in the intro cells
+    def add_header(self, data_asset_name, suite_name, batch_kwargs):
         self.add_markdown_cell(
-            f"""# Edit the Scaffolded Expectation Suite
+            f"""# Edit Your Expectation Suite
 Use this notebook to recreate and modify your expectation suite for:
-- **Data Asset**: `{data_asset_name}`
-- **Expectation Suite Name**: `{suite_name}`
 
```

Comment:
Is this bringing in old code again?

Reply:
I did not make this change - this is in feature/windowshopping

Reply:
Right -- I thought that @Aylr updated this in `feature/types`...if that's the case, then this PR would be asking to merge the *old* version back into `feature/types` (which we don't want). If not (and it was edited in `feature/windowshopping`), then this is the correct direction for the merge, but if that's the case, I'd be worried it's symptomatic of a git versioning issue we should look into a bit more closely.